pkgs/server/src/modules/names/index.ts

```diff
@@ -0,0 +1,2 @@
+export { NameService } from './service';
+export { Name } from './entity';
```

Comment:
Please reexport routes for this module.

Refer to: https://github.com/Jorgealvarezpb7/movies-watchlist/blob/main/pkgs/server/src/modules/images/index.ts